noir-projects/aztec-nr/aztec/src/macros/internals_functions_generation/abi_export.nr

```diff
@@ -2,10 +2,6 @@ use std::meta::type_of;
 
```

Comment:
Moved this file in this PR unlike the other funcitonality as the diff here is minimal and github managed to resolve it.